README.md

```diff
@@ -16,6 +16,8 @@ React Native Push Notification API for iOS.
 ### Install
 
 ```bash
+npm i @react-native-community/push-notification-ios --save
+or
 yarn add @react-native-community/push-notification-ios
 ```
```

Comment:
It would be great to split into two code blocks, so that it can easily be copied
````suggestion
Using npm:

```bash
npm i @react-native-community/push-notification-ios --save
```

or using Yarn:

```bash
yarn add @react-native-community/push-notification-ios
```
````